src/apprt/gtk/style.css

```diff
@@ -41,6 +41,22 @@ window.ssd.no-border-radius {
   border-radius: 0 0;
 }
 
+paned > separator {
```

Comment:
This shoud be part of the `paned > separator` section a little further down. We also need to tweak things to make wide tabs look a little more like they used to.

Reply:
Do you mean, move the block to be next to the `.terminal-window .notebook paned > separator` block? Or move the `margin` and `padding` settings within that block?

An argument for keeping them separate is that Gtk are looking at making this the default for all GtkPaned handles. If I move it into the existing block, then it will only apply to the separator for the terminal splits, not to all Paned separators. However, as this improves a general Gtk bug, does it make sense to apply to all separators, to prevent this issue affecting any other case where GtkPaned might be used?